<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigitalIODialog</class>
 <widget class="QDialog" name="DigitalIODialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>234</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>170</height>
      </size>
     </property>
     <property name="title">
      <string>Digital Outputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>8:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut8">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>6:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut6">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut2">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut1">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>5:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut5">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>7:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut7">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut4">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxDigitalOut3">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>170</height>
      </size>
     </property>
     <property name="title">
      <string>Digital Inputs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn1">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>5:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn5">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>8:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn8">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn2">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_14">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>6:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn6">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>3:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn3">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QLabel" name="label_23">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>7:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn7">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="minimumSize">
           <size>
            <width>21</width>
            <height>17</height>
           </size>
          </property>
          <property name="text">
           <string>4:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelDigitalIn4">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
